test/test_manifest.json

```diff
@@ -3589,6 +3589,14 @@
        "type": "eq",
        "nativeImageDecoderSupport": "none"
     },
+    {  "id": "decodeACSuccessive",
+       "file": "pdfs/decodeACSuccessive.pdf",
+       "md5": "7749c032624fe27ab8e8d7d5e9a4a93f",
+       "rounds": 1,
+       "link": true,
```

Comment:
This should be removed; the test case is in this commit itself.

Reply:
D'oh; good eye, fixed now!